starlette_admin/views.py

```diff
@@ -325,13 +329,28 @@ async def is_action_allowed(self, request: Request, name: str) -> bool:
         return True
 
     async def get_all_actions(self, request: Request) -> List[Optional[dict]]:
-        actions = []
         assert self.actions is not None
+        actions = []
         for action_name in self.actions:
             if await self.is_action_allowed(request, action_name):
                 actions.append(self._actions.get(action_name))
         return actions
 
+    async def get_actions(
```

Comment:
I believe we can extend the logic of get_all_actions(self, request: Request) and eliminate the need for this function.

Reply:
Can we add also a simple unit test [here](https://github.com/jowilf/starlette-admin/blob/main/tests/test_custom_actions.py) to make sure that future PR will not break the `exclude_actions_from_list` and `exclude_actions_from_details` params. I can also help if you authorize me to push here. Thanks

Reply:
@jowilf I added you as a collaborator to our repo. Feel free to push to this branch as needed!

Reply:
@jowilf I merged `get_actions` into `get_all_actions` and added tests for the excludes.

Good thing, because they caught a logical error that put forbidden actions on the detail page!

Reply:
> @jowilf I added you as a collaborator to our repo. Feel free to push to this branch as needed!
Thanks, but I was thinking about this ->
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork. It also allows the pre-commit bot the pre-commit bot to address linting issues.

Reply:
@jowilf I spent way too long looking for the button before googling it, but that button is not available for forks that live in a GitHub Organization: https://github.com/orgs/community/discussions/5634

You have write permissions for our fork, so if you need to make any further changes, you should be able to.